lib/credential/domain/use_cases/get_claims_use_case.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:polygonid_flutter_sdk/common/domain/domain_logger.dart';
 import 'package:polygonid_flutter_sdk/common/domain/entities/filter_entity.dart';
 import 'package:polygonid_flutter_sdk/common/domain/use_case.dart';
+import 'package:polygonid_flutter_sdk/constants.dart';
```

Comment:
Domain cannot have dependency of the constants file.
`GENESIS_PROFILE_NONCE` needs to be in Domain.